### DISCUSSION TITLE
How to fix this Auth Failed page.

### DISCUSSION THREAD

**Discussion by Anas099X** [pocketbase/pocketbase#4410] [Q&A] [answered]
Hello. I tried following the [document page](https://pocketbase.io/docs/authentication/) in setting up a OAuth Github login using Manual Code exchange method but I got this error
![image](https://github.com/pocketbase/pocketbase/assets/99427334/e7ea7745-80e8-4001-b171-d13cc7e3890b)

**Comment by ganigeorgiev** [comment, marked as answer]
If you are using the "Manual code exchange" flow then the redirect handler must be your own custom redirect page, not `/api/oauth2-redirect`.

If you need further help, please provide a sample code or at least elaborate a little on your setup.

**Comment by Anas099X** [comment]
Thanks! that worked
